### ISSUE TITLE
rule_account_disable_post_pw_expiration fails to remediate commented out value

### ISSUE THREAD

**Issue by dahaic** [OpenSCAP/scap-security-guide#2109]
File ```/etc/defaults/useradd``` with contents:
```
# useradd defaults file
GROUP=100
HOME=/home
#INACTIVE=35
EXPIRE=
SHELL=/bin/bash
SKEL=/etc/skel
CREATE_MAIL_SPOOL=yes
```
won't be remedied.

**Comment by yuumasato** [comment]
I was in a situation when
```
INACTIVE=-1 
```
would not be remedied.

In my case it is related to #1420

**Comment by yuumasato** [comment]
@dahaic a fix for `replace_or_append` has been merged (#2107), could you verify if this issue is still valid?

**Comment by yuumasato** [comment]
I can try it actually
